rotkehlchen/externalapis/etherscan.py

```diff
@@ -285,11 +289,56 @@ def get_transactions(
             for entry in result:
                 gevent.sleep(0)
                 try:
-                    tx = deserialize_ethereum_transaction(  # type: ignore
-                        data=entry,
-                        internal=is_internal,
-                        ethereum=None,
-                    )
+                    # Handle genesis block transactions
+                    if is_genesis_tx(entry) is False:
+                        tx = deserialize_ethereum_transaction(  # type: ignore
+                            data=entry,
+                            internal=is_internal,
+                            ethereum=None,
+                        )
+                    else:
+                        # Handling genesis transactions
+                        if self.db is None:
+                            raise DeserializationError('DB not initialized')
```

Comment:
```suggestion
```

DB is always initialized in this class. It's a constructor argument and it's non optional.

Reply:
Oops, missed it